examples/ulx3s/Makefile

```diff
@@ -1,10 +1,28 @@
-all: blinky.bit
+all: blinky.bit hello.bit
 
-blinky.json: blinky.v
-	yosys -p "synth_ecp5 -json blinky.json" blinky.v
+hello.json: pll_120.v
 
-blinky_out.config: blinky.json
-	nextpnr-ecp5 --json blinky.json --textcfg blinky_out.config --45k
+IDCODE ?= 0x21111043 # 12f
```

Comment:
Perhaps we should have a device option here too, that would also change the device passed to nextpnr?

Reply:
It looks like only (?) the -12f part requires this option.  I'm not sure where the right place to fix this is (especially in light of things like https://github.com/SymbiFlow/prjtrellis/issues/55#issuecomment-452689924 ). Should nextpnr-ecp5 know about the 12f and its special id instead?